zerver/models/realms.py

```diff
@@ -661,6 +664,7 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
         email_changes_disabled=bool,
         emails_restricted_to_domains=bool,
         enable_guest_user_indicator=bool,
+        enable_guest_user_dm_warning=bool,
```

Comment:
Just a habits thing: Generally when adding things to lists like this, you want to keep them sorted, which means putting this one line up.

Reply:
Fixed it!